src/app/stylesheets/variables.less

```diff
@@ -83,6 +83,10 @@
 @table-bg-active:               @table-bg-hover;
 @headings-color:                @text-main-color;
 @mno-row-sublink-color:         @text-main-strong-color;
+@mno-feedback-bg-color:         @mnoe-admin-panel-dark-white;
+@mno-comment-bg-color:          @mnoe-admin-panel-white;
+@mno-question-bg-color:         @mnoe-admin-panel-white;
+@mno-answer-bg-color:           @mnoe-admin-panel-dark-white;
```

Comment:
You can't use project named color variables in `variables.less`. Only generic ones, like `@bg-main-color`, etc.